sdks/cpp/common/src/ParamAccessor.cpp

```diff
@@ -514,7 +514,7 @@ void ParamAccessor::setValue(const std::string& peer, const Value &src, ParamInd
             setterAt[value.kind_case()](value, src, idx);
         } else {
             // update scalar value or whole array
-            value.CopyFrom(src);
+            value = src;
```

Comment:
did you check that operator= just calls CopyFrom?

Reply:
It does just call CopyFrom. I will change it back though just to make it a bit more clear.